frontend/src/components/MapView/MapView.tsx

```diff
@@ -77,7 +78,7 @@ export let MapView = withStore(
   let providers = store.get('providers')
   let selectedRepresentativePoint = store.get('selectedRepresentativePoint')
   let selectedProvider = store.get('selectedProvider')
-  const pointGeoJson = store.get('pointGeoJson')
+  let pointFeatureCollections = store.get('pointFeatureCollections')
```

Comment:
y u change from const to let 😡

Reply:
Is there a diff in behavior / performance? Enlighten me

Reply:
`let` variables are mutable, `const` variables are not. it's good practice generally to make things consts because it prevents accidentally making changes that cause unexpected changes in behaviour. i like to declare everything as a const and then if it ends up needing to be mutable i'll go back and change it to a let.

Reply:
it's sort of related to how in OOD you don't want to just make member functions public - it's better to start with everything private and then expose the behaviours you want

Reply:
(this is not really important so don't feel the need to submit changes to this PR based on this comment - just a useful rule of thumb moving forwards)